config.toml.example

```diff
@@ -0,0 +1,60 @@
+[flaskapp]
+DEBUG = true
+TESTING = true
+PROPAGATE_EXCEPTIONS = ""
+SECRET_KEY = "10505aec23775f6ecb61ded347654282d1be2c673e3b37b0372db6557f39f0ca"
```

Comment:
Pretty sure someone from gitleaks is going to complain about this secret-looking string in a secret-looking variable ;-)
Maybe set something like "change-me" ?